### ISSUE TITLE
Multiple routes

### ISSUE THREAD

**Issue by nobleach** [madeagency/reactivity#82]
This appears to be locked in to one universal route (in Express) All traffic flows through `render.js` What if one wanted to do multiple routes? (not using React router) I'm considering pulling it all apart and playing with it. My app would have to do some configuration on startup - instead of doing all the work on req/res. I just didn't know if you might have some immediate thoughts.

**Comment by rocbear** [comment]
@nobleach I'm not sure I fully understand your question but I'll try and point you in the right direction. `render.js` is compiled to `build/server.js` which is used as middleware in the express app defined in [src/server/index.js](https://github.com/madeagency/reactivity/blob/master/src/server/index.js). You can see this happening in [bin/server.prod.js](https://github.com/madeagency/reactivity/blob/master/bin/server.prod.js).

In short, if you'd like to add routes at the express level, the best place to do it would probably be `src/server/index.js`.

Hope this helps!

**Comment by rocbear** [comment]
No problem, glad I could help clear things up.
